src/index.js

```
@@ -175,7 +175,7 @@ export function createRPCHandler({
         delete error.stack;
         error.initialArgs = args;
         store.dispatch(actions && actions.failure(error));
-        return e;
+        throw e;
```

@KevinGrandon @lhorie This seems like a breaking change. Do we plan on updating`fusion-plugin-rpc-redux-react` to catch exceptions by default? Otherwise all calls using `withRPCRedux` need to explicitly handle exceptions at the call site. Previously you only needed to worry about exceptions in the reducer.

I think ideally we'd default to discarding the exception since the common use case is to handle in the reducer and provide an option to propagate exceptions.

---

I don't believe this is necessarily a breaking change, because 1) I don't believe resolving to an error was correct behavior and 2) this code could already throw before:

https://github.com/fusionjs/fusion-rpc-redux/blob/d522c10652aa6c57538266720ac46317b3248c5a/src/index.js#L169

---

I guess it's technically not a breaking change in `fusion-rpc-redux` I believe it will cause a regression in `fusion-plugin-rpc-redux` do to its assumption that the failure will be handled in the redux reducer and doesn't have to be explicitly handled at the call site.

For example, the fusion docs recommend fetching data for a component as follows:

```
compose(
  withRPCRedux('fetchData'),
  connect({myData} => {myData}),
  prepare(props => {
    const {myData, fetchData} = props;
    if (myData) {
      return Promise.resolve();
    }
    return fetchData();
  })
)(MyComponent);
```

Previously, every time the sideEffect was run, an exception would emitted a failure action for `fetchData` that could be handled with `createRPCReducer`'s failure case.

With this code change, if `fetchData` fails, we'll get an unhandled promise exception whenever the sideEffect is triggered in [prepared.js](https://github.com/fusionjs/fusion-react/blob/master/src/async/prepared.js#L42).

Does that make sense?